components/raftstore-v2/src/tablet.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::sync::{
+    atomic::{AtomicU64, Ordering},
+    Arc, Mutex,
+};
+
+struct LatestTablet<EK> {
+    data: Mutex<Option<EK>>,
+    version: AtomicU64,
+}
+
+/// Tablet may change during split, merge and applying snapshot. So we need a shared value to
```

Comment:
When can it use a stale tablet rather than latest

Reply:
For example, it there is no on-going admin commands, raftstore can safely assume cached tablet is latest, so no need to refresh cache.

Reply:
oh, it's just to avoid calling `lock()`